part_07_Transfer_Learning_and_Tensorhub/README.md

```diff
@@ -0,0 +1,20 @@
+# Part #7: Transfer Learning & TensorHub
+## Table of Content
+- Transfer learning using tf.keras
+- Feature Extraction
+- Fine Tuning
+- Transfer learnign using TensorHub
```

Comment:
Great! An excellent way to start.

Additionally, I was thinking about another two topics that we may want to include. First, I think it would be necessary to add how we can train a model, and then use it in another model (with fixed or trainable weights). Secondly, we should also mention how we can extract learned weights from a pre-trained model, which is particularly useful in visualization and interpretation.

Reply:
i will add them both